checker/src/block_visitor.rs

```diff
@@ -735,6 +729,27 @@ impl<'block, 'analysis, 'compilation, 'tcx, E>
         self.bv.get_u128_const_val(val)
     }
 
+    pub fn get_int_const_val(&mut self, mut val: u128, ty: Ty<'tcx>) -> Rc<AbstractValue> {
```

Comment:
A comment about the function will be most welcome.

Reply:
Added a method comment.